src/pages/library/dataBrowser-utils.js

```diff
@@ -80,7 +80,7 @@ const normalizeDataset = dataset => {
     dataReleasePolicy,
     contacts, curators, contributorNames,
     dataType, dataModality,
-    access: _.intersection(dataset.roles, ['reader', 'owner']).length > 0 ? datasetAccessTypes.GRANTED : datasetAccessTypes.CONTROLLED
+    access: dataset.accessLevel === 'reader' || dataset.accessLevel === 'owner' ? datasetAccessTypes.GRANTED : datasetAccessTypes.CONTROLLED
```

Comment:
```suggestion
    access: (dataset.accessLevel === 'reader' || dataset.accessLevel === 'owner') ? datasetAccessTypes.GRANTED : datasetAccessTypes.CONTROLLED
```

Is there any reason we don't have the same code as before but using `dataset.accessLevel` instead? Is there any issue with operator precedence using the `w || x ? y : z` construction?

Reply:
Nope, parens are fine to use here. There shouldn't be issue with operator precedence, but it does seem easier to read this way.